<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>API Warning</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/mc_mod_updater_icon.ico</normaloff>../icon/mc_mod_updater_icon.ico</iconset>
  </property>
  <widget class="QLabel" name="explanationLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>630</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CurseForge links require an API key to get data from the website.&lt;br/&gt;Modrinth links don't need an API key.&lt;br/&gt;Skipping this step means that you won't be able to use CurseForge links&lt;/p&gt;&lt;p&gt;To get an API key, visit this link: &lt;a href=&quot;https://console.curseforge.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://console.curseforge.com/&lt;br/&gt;&lt;/span&gt;&lt;/a&gt;1. Login or create an account&lt;br/&gt;2. Once you're logged in, go to the tab 'API keys' and copy the key&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="apiLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>62</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>API Key:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="apiInput">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>190</y>
     <width>551</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>631</width>
     <height>28</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close|QDialogButtonBox::Ignore|QDialogButtonBox::Save</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
